### PR TITLE
Update release page with times and details of 4.3.0

### PR DESCRIPTION
**Pull request by diegopetrucci**

### CODE REVIEW

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ The release process starts when the first build is provided to QA and ends when
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: Execution: `1h 54min` Cumulative: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: TBD`<br> `CNSMR-2338: TBD`<br> `CNSMR-2337: TBD`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```
Comment (dmcrodrigues) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: > Automated: Execution: 1h 54min Cumulative: 8h 41min

What the meaning of `Cumulative`? And why are we splitting this now?

Comment (diegopetrucci) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: This is the new format as reported by QA https://babylonpartners.atlassian.net/wiki/spaces/QA/pages/1210450470/QA+Sign+off+notes+iOS+4.3.0+Release+Babylon+25th+July+2019

Maybe @JasonDobo can shed some light on it. There was some discussion on the release channel but I am not sure if I can post it here.

Comment (JasonDobo) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: This is from me @dmcrodrigues previously I would total all the time for running the UI tests into a single number as if they ran on 1 machine with 1 simulator. This was a manual activity if you look at CircleCI i.e. https://circleci.com/workflow-run/a45d0d4d-77ae-46e0-a8fe-5b62f5461bb2 it just gives the time taken from start to finish.

As I don't want to manually calculate this anymore (a mistake on my part that I did it) I was just going to take what CircleCI reported, or would you like me to continue with this manual activity? Also Android report the time same as CircleCI which is start to finish and don't total all tests together as I was doing.

Comment (JasonDobo) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: Rui request I put both in for now so we can track if things are getting worse or better, hopefully then only putting in execution time

Comment (JasonDobo) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: TBD`<br> `CNSMR-2338: TBD`<br> `CNSMR-2337: TBD`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```

Comment (dmcrodrigues) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: I think we should keep both for a while until we have a history for the new number and we can still co-related with the old for a while but can I suggest new names for this? Maybe something like: `Parallel Execution` and `Serial Execution`?

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 2]
```diff
@@ -106,6 +106,7 @@ The release process starts when the first build is provided to QA and ends when
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: TBD`<br> `CNSMR-2338: TBD`<br> `CNSMR-2337: TBD`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```
Comment (JasonDobo) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: Execution: `1h 54min` Cumulative: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: TBD`<br> `CNSMR-2338: TBD`<br> `CNSMR-2337: TBD`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```

File: Cookbook/Technical-Documents/ReleaseProcess.md (hunk 1 above)
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: Parallel execution: `1h 54min` Serial execution: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: TBD`<br> `CNSMR-2338: TBD`<br> `CNSMR-2337: TBD`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```

Comment (JasonDobo) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: Nice idea. @diegopetrucci Can we approve this PR

Comment (diegopetrucci) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: It's missing your times (last three tickets). Would you mind adding them? you can do via the suggestion thingy

Comment (JasonDobo) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: I added it to the slack thread, sorry missed your reply

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 3]
```diff
@@ -106,6 +106,7 @@ The release process starts when the first build is provided to QA and ends when
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: Parallel execution: `1h 54min` Serial execution: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: TBD`<br> `CNSMR-2338: TBD`<br> `CNSMR-2337: TBD`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```
Comment (JasonDobo) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated: Parallel execution: `1h 54min` Serial execution: `8h 41min`<br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: 1.5hours`<br> `CNSMR-2338: 30min`<br> `CNSMR-2337: 1hour`<br>  | Total: **TBDh TBDmin** | 22.07.2019 | 30.07.2019 |
```

**Review by MichalTKwiecien — APPROVED** [review]

**Review by brocoo — APPROVED** [review]

**Pull request merged**